applications/minotari_ledger_wallet/wallet/Cargo.toml

```
@@ -39,7 +39,7 @@ default = []
 pending_review_screen = []
 
 [package.metadata.ledger]
-curve = ["ed25519"]
+curve = ["ed25519", "secp256k1"]
```

```suggestion
curve = [ "secp256k1"]
```
Can we remove ed25519 or do we still use that?